omod/src/main/webapp/resources/scripts/custom/deletePatient.js

```diff
@@ -0,0 +1,66 @@
+var delPatient = delPatient || {};
+var jqObject = $();
+
+delPatient.deletePatientCreationDialog = null;
+
+/**
+ * Functions used to Delete a Patient
+ */
+
+delPatient.returnUrl = "/coreapps/findpatient/findPatient.page?app=coreapps.findPatient";
+
+delPatient.createDeletePatientCreationDialog = function() {
+    delPatient.deletePatientCreationDialog = emr.setupConfirmationDialog({
+        selector: '#delete-patient-creation-dialog',
+        actions: {
+            confirm: function() {
+                var deleteReason = jQuery('#delete-reason').val().trim(); //Retrieve text from text box
+                if(deleteReason && deleteReason.length > 0) { //Should not be invalid or empty
+                    jq.ajax({
+                        url: '/openmrs/ws/rest/v1/patient/'+delPatient.patientUUID,
+                        type: 'DELETE',
+                        body: {
+                            "void_reason": deleteReason
+                        },
+                        success: function() {
+                            jqObject.toastmessage( 'showToast', { type: 'success',
```

Comment:
Yeah that's what I had done in the first place. I'll revert back to it, with the translated message